test/tests/outputs/checkpoint/checkpoint_child.i

```diff
@@ -40,7 +40,7 @@
 
 [Executioner]
   type = Transient
-  num_steps = 11
+  num_steps = 2
```

Comment:
why that change?

Reply:
I m fine with it, just curious

Reply:
Basically so the test doesn't take as long. By changing the multiapp type from transient to full solve, we are running all time steps of the sub app after every master app time step. Not necessary or desirable.